packages/landing/src/components/layer.switcher.dropdown.tsx

```diff
@@ -60,7 +58,7 @@ export class LayerSwitcherDropdown extends Component<unknown, LayerSwitcherDropd
   render(): ComponentChild {
     if (this.state.isEnabled !== true) return;
     return (
-      <div>
+      <div class="LuiDeprecatedForms">
```

Comment:
Deprecated feature?

Reply:
Yup, this is unfortunately the easiest way to get this working atm :(